src/style.css

```
@@ -1,3 +1,80 @@
+* {
+  box-sizing: border-box;
+}
+
+html{
+  font-family: Roboto, sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+:root {
+  --blue-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: space-between;
+}
+
+img {
+  display: block;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: #000;
+  display: block;
+  padding: 23px 0;
+  text-transform: uppercase;
+  font-weight: 500;
+  line-height: 14.06px;
```

will it work? it's better to use integers
```suggestion
  line-height: 14px;
```